contracts/volt/VoltV2.sol

```diff
@@ -0,0 +1,528 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity =0.8.13;
+
+import {CoreRef} from "../refs/CoreRef.sol";
+import {TribeRoles} from "../core/TribeRoles.sol";
+
+// Forked from Uniswap's UNI
+// Reference: https://etherscan.io/address/0x1f9840a85d5af5bf1d1762f925bdaddc4201f984#code
+
+contract VoltV2 is CoreRef {
+    /// @notice EIP-20 token name for this token
+    // solhint-disable-next-line const-name-snakecase
+    string public constant name = "Volt";
+
+    /// @notice EIP-20 token symbol for this token
+    // solhint-disable-next-line const-name-snakecase
+    string public constant symbol = "VOLT";
+
+    /// @notice EIP-20 token decimals for this token
+    // solhint-disable-next-line const-name-snakecase
+    uint8 public constant decimals = 18;
+
+    /// @notice Total number of tokens in circulation
+    // solhint-disable-next-line const-name-snakecase
+    uint256 public totalSupply = 1_000_000_000e18; // 1 billion Volt
+
+    /// @notice Address which may mint new tokens
+    address public minter;
```

Comment:
Let's pull in CoreRef and allow the minter role to mint, and remove the `minter` address.